libs/zzz/consts/src/common.ts

```diff
@@ -31,21 +32,39 @@ export const allStatKeys = Array.from(
 )
 export type StatKey = (typeof allStatKeys)[number]
 
-export const allElementalKeys = [
+const extraPandoStatKeys = [
+  'frost_dmg_',
+  'impact',
+  'anomMas',
+  'dmg_',
+  'shield_',
+] as const
+export const allPandoStatKeys = Array.from(
+  new Set([
+    ...allDiscMainStatKeys,
+    ...allDiscSubStatKeys,
+    ...extraPandoStatKeys,
+  ])
+)
+export type PandoStatKey = (typeof allPandoStatKeys)[number]
+
+export const allAttributeKeys = [
   'electric',
   'fire',
   'ice',
   'physical',
   'ether',
+  'frost',
 ] as const
-export type ElementalKey = (typeof allElementalKeys)[number]
+export type AttributeKey = (typeof allAttributeKeys)[number]
 
 export const allAttributeDamageKeys = [
   'electric_dmg_',
   'fire_dmg_',
   'ice_dmg_',
   'physical_dmg_',
   'ether_dmg_',
+  'frost_dmg_',
```

Comment:
The addition here also affects the UI:
<img width="931" alt="image" src="https://github.com/user-attachments/assets/4026826c-8f4e-43a7-8b02-8e6a84ef6541" />
Since "frost" doesn't currently exist, we might have to create `allAttributeDamageKeysAndFrost`

Reply:
we can move frost to TODO tbh

Reply:
yeah let's do that, I value keeping prod consistent for now.